server/controllers/reports_proposed/reports.js

```diff
@@ -74,6 +80,7 @@ exports.build = function (req, res, next) {
 
   var handler = documentHandler[target]; //handler will contain a object with two property, template for structure and context for data
   var options = req.body;
+  options.stylePath = path.join(__dirname);
```

Comment:
This can be `options.stylePath = __dirname`;  Although, I would just name it `path` or something, since it doesn't have anything to do with styling (even if you use it later for styling).